bundles/net.tourbook/src/net/tourbook/ui/views/rawData/ActionDeleteTourValues.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (C) 2021 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.ui.views.rawData;
+
+import net.tourbook.Messages;
+import net.tourbook.common.util.ITourViewer3;
+import net.tourbook.tour.TourManager;
+
+import org.eclipse.jface.action.Action;
+import org.eclipse.swt.widgets.Display;
+
+public class ActionDeleteTourValues extends Action {
```

Comment:
The delete action is disabled but other tour actions are enabled

![mt delete action disabled](https://user-images.githubusercontent.com/1283445/115114484-f8e79b80-9f8f-11eb-95d0-ac8361fefce5.png)

Reply:
I've disabled it because it doesn't seem to work when selecting a month category.
When I try to re-import tours like in your screenshot, it says no tours were selected and nothing happens:
![image](https://user-images.githubusercontent.com/25235665/115314770-199c2680-a133-11eb-80d2-589d51d99d60.png)

Reply:
It works when the `A` button is selected, you can see the `A` button at the top of my above screenshot. I introduced the `A` button to easily select all tours in a week/month/year at once. `A` is the shortcut for ALL

Reply:
Oh ok, it is not intuitive but it works. I have fixed it.

![image](https://user-images.githubusercontent.com/25235665/115410206-ac7aa680-a1af-11eb-9371-17a2836ca997.png)

Reply:
>Oh ok, it is not intuitive but it works. 

The tooltip of the `A` button tells more

It's often a question, why is an action disabled and how to enable it. For some actions it obvious but for some not.